client/src/telemetry/metrics.yaml

```
@@ -0,0 +1,67 @@
+---
+# Schema
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+page_view:
+  path:
+    type: url
+    lifetime: application
+    send_in_pings:
+      - page
+      - action
+    description: |
+      The URL path of the page that was viewed.
+    data_sensitivity:
+      - web_activity
+    bugs:
+      - "https://bugzilla.mozilla.org/show_bug.cgi?id=1782509"
```

```suggestion
      - https://bugzilla.mozilla.org/show_bug.cgi?id=1782509
```

I don't think you need double quotes here